hw_5_task_1.py

```diff
@@ -0,0 +1,14 @@
+# 1
+# there is a space-separated string with the names "john peter brian Morgan Adam Maria bart"
+# convert the string so that each name starts with a capital letter.
+
+string_to_mod = 'john peter brian Morgan Adam Maria bart'
+modified_list = []
+
+for word in string_to_mod.split(' '):
+    modified_list.append(word.capitalize())
+
+
+modified_string = ' '.join(modified_list)
+
+print(modified_string)
```

Comment:
it works, but over engineering))

Python already did it for you in some specific function of string